### DISCUSSION TITLE
X chromosome simulation

### DISCUSSION THREAD

**Discussion by AliPearson** [tskit-dev/msprime#2171] [General]
I looking to simulate the X chromosome, possible under a fairly complex demographic model including both males and females. Is there a trick with msprime to do this?

**Comment by jeromekelleher** [comment]
This isn't something we've done properly unfortunately @AliPearson - it's probably something that should be documented, at a minimum. The best discussion I'm aware of is here: https://github.com/popsim-consortium/stdpopsim/issues/383

Please do let us know if you have any success with this!
